### ISSUE TITLE
libusb1 - enable build with examples

### ISSUE THREAD

**Issue by realsnick** [NixOS/nixpkgs#207381]
### Describe the bug
require to build `libusb1` with the examples `(BUILD_EXAMPLES)` - specificaly `fxload` binary as it supports usb3 firmwares

### Expected behavior
Have `fxload` binary available for use

### Additional context
there is already a package for `fxload` though not supporting usb3, in addition this binary source code already exists ,actual source code  of `fxload` moved to `libusb` under `examples/fxload.c`. 
https://github.com/libusb/libusb/blob/master/examples/fxload.c

https://github.com/NixOS/nixpkgs/issues/200152

**Comment by realsnick** [comment]
the examples are now building but the `fxload` binary is actually a libtool script...will appreciate any insights on how to fix it

**Comment by realsnick** [comment]
libusb1 - enable build with examples #207513

**Comment by realsnick** [comment]
@dasJ why did you close the PR?

**Comment by dasJ** [comment]
Sorry I usually don't explain it anymore. You messed up a staging rebase (which is fairly easy to get wrong tbh), causing lots of people to be subscribed to the (for them irrelevant) PR. There is no way you can unsubscribe others which causes loads of people to get notified whenever you do anything to the PR.
The only solution is to close it and create a new PR that goes directly to the correct branch or where you rebase properly.

You can check the proper rebase instructions in the CONTRIBUTING.md in the root of the repository. It's not very obvious but it's the only way to prevent unnecessary pings unfortunately.

**Comment by realsnick** [comment]
i was trying to follow these instructions, sorry for messing it up... 
Should i just create my branch out of the staging branch and create a new PR for the staging branch?

**Comment by dasJ** [comment]
No need to apologise, it's really common.

If you want to be extra careful in the future, you can mark the PR as a draft before doing all that and marking it as ready once everything is done :)

**Comment by realsnick** [comment]
thanks, that is a great idea! didn’t realize there is a way to make a draft PR.

**Comment by realsnick** [comment]
merged
